recipes/w4mclassfilter/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "0.98.15" %}
-{% set md5 = "10e22b263ce241e8c128a53223b000fe" %}
+{% set version = "0.98.16" %}
+{% set sha256 = "0610bf061ab09c23a6d8dd0c7245fedd4a40158ef36b76390e04ef5858233fc4" %}
 
 package:
   name: w4mclassfilter
   version: {{ version }}
 
 source:
   url: https://github.com/HegemanLab/w4mclassfilter/releases/download/v{{ version }}/w4mclassfilter_{{ version }}.tar.gz
-  md5: {{ md5 }}
+  sha256: {{ sha256 }}
 
 build:
   noarch: generic
-  number: 0
```

Comment:
Since the version is updated, the build number should be 0, and not 1.

Reply:
If I change it to zero, it no longer will build: CircleCI said (correctly) that I had already submitted a build with build 0 - I discovered after creating the pull request that I was supposed to use a sha256 hash, but when I resubmitted, CircleCI said that I had already done a build and needed to increment the build number.  

Are you suggesting that if I have a build failure (or success) I have to change my package version? If so (and I want the number to reveal the version of the content), would "v0.98.16-1" be a satisfactory alternative package version name?